### PR TITLE
JavaDocs.

### PR DESCRIPTION
**Pull request by aemogie**

### CODE REVIEW

**Comment by aemogie** [comment]
image
WIP and nae nae

File: src/main/java/graphics/Window.java [hunk 1]
```diff
@@ -15,197 +15,298 @@
...
+ */
 public class Window {
-
+    
+    /**
+     * SceneManager. Need a better explanation? The thingy that does things with scenes.
+     */
     private SceneManager sceneManager;
-
+
```
Comment (aemogie) [inline src/main/java/graphics/Window.java:27]: Ignore these, I have "Keep indents on empty lines" on, for easier navigation :)

File: src/main/java/util/Engine.java [hunk 2]
```diff
@@ -6,73 +6,123 @@
...
-
+    
+    /**
+     * The exact time the engine started in milliseconds.
+     */
     private final long startMillis;
+    
+    /**
+     * No clue
+     */
```
Comment (athaun) [inline src/main/java/util/Engine.java:26]: This is a reference to the Window class, which contains data like the glfw window, width, height, delta time and the main loop.

File: src/main/java/util/Engine.java [hunk 3]
```diff
@@ -6,73 +6,123 @@
...
     private Window window;
+    
+    /**
+     * Whether the {@code Engine} is running.
+     */
```
Comment (athaun) [inline src/main/java/util/Engine.java:31]: image

Comment (athaun) [reply src/main/java/util/Engine.java:31]: this seems clear that it is about whether or not the engine is running

File: src/main/java/util/Engine.java [hunk 4]
```diff
@@ -6,73 +6,123 @@
...
     private boolean running;
+    
+    /**
+     * The time between last frame and current
+     */
     private float deltaTime;
-
+    
     //private to prevent creating new instances
+    /**
```
Comment (athaun) [inline src/main/java/util/Engine.java:40]: this is generally the purpose of `private`, there is no need to comment this.

**Review by athaun — CHANGES_REQUESTED** [review]
Please look through my comments, some of the javadocs are unnecessary.
I want to refrain from merging this until other PRs are closed to avoid unnecessary merge conflicts.

File: src/main/java/util/Engine.java (hunk 4 above)
Comment (aemogie) [reply src/main/java/util/Engine.java:41]: the comment was already there, i just made it to a javadoc. will remove it 👍

File: src/main/java/util/Engine.java (hunk 2 above)
Comment (aemogie) [reply src/main/java/util/Engine.java:27]: oh ye, forgot i did that. was making a joke to prove my documenting skills xD
will fix that

**Pull request closed without merging**